etc/kworkflow.config

```diff
@@ -0,0 +1,29 @@
+# Default configuration file
```

Comment:
I would like to add a comment here saying that no variables should be removed from this file, in order to kw work properly, just to warn users that want to customize their global configuration files.

Reply:
I did not add this, because this is not true. For example, if the user uses libvirt as a virtualized, the QEMU command has no effect. However, now I'm think about adding some hardcoded variable just in case that a variable was not loaded from the config file. After this PR we could work to make kw stable and generic from the point of view of the configuration variables.